katportalclient/test/test_client.py

```diff
@@ -1046,7 +1045,7 @@ def test_sensor_history_single_sensor_without_value_time(self):
         # ensure time order is increasing
         time_sec = 0
         for sample in samples:
-            self.assertGreater(sample.sample_time, time_sec)
+            self.assertGreater(float(sample.sample_time), float(time_sec))
```

Comment:
As discussed - this cast is unexpected.  `sample_time` should already be a float.  Check implementation and test input.